simulator/src/resim/mod.rs

```diff
@@ -86,25 +86,27 @@ pub enum Command {
 pub fn run() -> Result<(), Error> {
     let cli = ResimCli::parse();
 
+    let mut out = std::io::stdout() ;
+
     match cli.command {
         Command::CallFunction(cmd) => cmd.run(),
         Command::CallMethod(cmd) => cmd.run(),
-        Command::ExportAbi(cmd) => cmd.run(),
-        Command::GenerateKeyPair(cmd) => cmd.run(),
+        Command::ExportAbi(cmd) => cmd.run(&mut out),
+        Command::GenerateKeyPair(cmd) => cmd.run(&mut out),
         Command::Mint(cmd) => cmd.run(),
-        Command::NewAccount(cmd) => cmd.run(),
+        Command::NewAccount(cmd) => cmd.run(&mut out),
         Command::NewBadgeFixed(cmd) => cmd.run(),
         Command::NewBadgeMutable(cmd) => cmd.run(),
         Command::NewTokenFixed(cmd) => cmd.run(),
         Command::NewTokenMutable(cmd) => cmd.run(),
```

Comment:
Is there a particular reason we're using `writeln` for some but not all?

Reply:
Yes. The other ones don't produce any output - so we'd just be passing the IO parameter for nothing

Reply:
That's perfect! Thanks for clarifying.